packages/rest/src/rest-server.ts

```diff
@@ -595,6 +599,7 @@ export class RestServer extends Context implements Server {
 export interface RestServerConfig {
   host?: string;
   port?: number;
+  basePath?: string;
```

Comment:
I don't see any code to use server-level `basePath`? Is it to be implemented?

Reply:
It will be a part of a subsequent commit in this PR, but it'll need to wait until @jannyHou lands #916